<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#222629;
QLineEdit{
	background:transparent;
	color:#F2F2F2;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>360</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
font-weight:600;
font-size:16px;</string>
    </property>
    <property name="placeholderText">
     <string> URL link</string>
    </property>
   </widget>
   <widget class="QComboBox" name="type">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>170</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#F2F2F2;</string>
    </property>
    <item>
     <property name="text">
      <string>video</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>audio</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="browse">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>120</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
color:#F2F2F2;</string>
    </property>
    <property name="text">
     <string>Brows</string>
    </property>
   </widget>
   <widget class="QComboBox" name="res">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>70</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
color:#F2F2F2;</string>
    </property>
    <item>
     <property name="text">
      <string>720p</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>360p</string>
     </property>
    </item>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>360</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
color:#F2F2F2;</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="download">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>530</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
color:#F2F2F2;</string>
    </property>
    <property name="text">
     <string>Download</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>360</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
font-weight:600;
font-size:16px;</string>
    </property>
    <property name="placeholderText">
     <string> Save location</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>360</width>
      <height>41</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
font-weight:600;
font-size:16px;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Download status</string>
    </property>
   </widget>
   <widget class="QComboBox" name="single_milti">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
color:#F2F2F2;</string>
    </property>
    <item>
     <property name="text">
      <string>Single Download</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Playlist Download</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>604</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
